src/components/profile/newIVAsModal/newIVAModal.tsx

```diff
@@ -151,25 +151,32 @@ const NewIVAModal = (props: NewIVAModalProps) => {
                 : `Please enter your ${promptText}:`}
             </p>
             <input
-              type="text"
+              type={
+                clickedButton === IVAType.Fax || clickedButton === IVAType.Phone
+                  ? "tel"
+                  : "text"
+              }
               id="iva"
               name="iva"
               className="mb-4 w-100"
               required={true}
             />
             <p>
-              We will send a verification code to this address after you request
-              verification on your profile page.
+              <strong>
+                In order to verify your address, you will need to request
+                verification from your profile page.
+              </strong>
```

Comment:
So your solution to GSI-756 is to say this louder and to change the button text?

Not sure if that's sufficient but let's do it like that for now.